### ISSUE TITLE
Problems enabling rule `wrapping-the-expression-body-of-a-function` via `.editorconfig`

### ISSUE THREAD

**Issue by TribuneX** [pinterest/ktlint#2032]
## Expected Behavior
I cannot get the rule [https://pinterest.github.io/ktlint/0.49.1/rules/configuration-ktlint/#wrapping-the-expression-body-of-a-function](wrapping-the-expression-body-of-a-function) to work with my `.editorconfig`:

I do not want to enable all experimental rules, but just this one. That is the relevant part of my current `.editorconfig`:

```
[{*.kt,*.kts}]
ktlint_experimental_function-signature = enabled
ktlint_function_signature_body_expression_wrapping = always
[...]
```

## Observed Behavior
The following code should violate the rule, but does not:
```kotlin
class Example {

    fun functionInOneLine(number: Number): String = "my string"
}
```

Only if I add the additional option `ktlint_experimental = enabled` to my `.editorconfig`, the file is correctly formatted to:

```kotlin
class Example {

    fun functionInOneLine(number: Number): String =
        "my string"
}
```

## Steps to Reproduce
See examples above.

## Your Environment
macOS `13.3.1` with ktlint `0.49.1`

**Comment by TribuneX** [comment]
Seems that I need to set `ktlint_standard_function-signature = enabled` instead, although this rule is listed as an [experimental rule](https://pinterest.github.io/ktlint/0.49.1/rules/experimental/#function-signature)?

**Comment by paul-dingemans** [comment]
> Seems that I need to set `ktlint_standard_function-signature = enabled` instead, although this rule is listed as an [experimental rule](https://pinterest.github.io/ktlint/0.49.1/rules/experimental/#function-signature)?

There was already a kind of warning on top of the Experimental Rules documentation:
![Screenshot 2023-05-17 at 18 10 40](https://github.com/pinterest/ktlint/assets/5195292/df5cf4c5-cbeb-41f0-872b-8bd85a7a0720)

Would it be more clear if I change the last line this message as follows?

> Or, when a specific experimental rule is provided in rule set with id `rule-set-id`, it can be enabled via .editorconfig property `ktlint_<rule-set-id>_<rule-id>`. 

I will change all rule ids on the experimental and standard rules pages to the fully qualified rule id.


> This is not clear to me reading the docs:
>
> > This setting only takes effect when rule function-signature is enabled.
> Maybe its worth adding an example how to enable this?

This means that the experimental rule `function-signature` needs to be enabled. Either enable all experimental rules or enable this specific rule.

**Comment by TribuneX** [comment]
I think my first confusion is mostly because it was not clear if the rule `ktlint_function_signature_body_expression_wrapping` is a standard rule or an experimental rule, since its listed under `KtLint configuration`. I guess its neither of both. 
However, I did not know how to translate this advice into a `.editorconfig`:
> This setting only takes effect when rule function-signature is enabled.

Maybe just add the following:
> This setting only takes effect when rule function-signature is enabled (e.g. via `ktlint_standard_function-signature = enabled` in your `.editorconfig`).

The second confusion was because of this sentence in the important box:
> Or, when a specific experimental rule is enabled via `.editorconfig` property `ktlint_<rule-set-id>_<rule-id>`.

I am missing some information about what the `rule-set-id` actual is. I though `function-signature` is the `rule-id` and since this is an experimental rule `experimental` would be the `rule-set-id`. But its `ktlint_standard_function-signature` instead.

Is that because the first sentence in the box states?
> Experimental rules in ktlint are part of the [standard ruleset](https://github.com/pinterest/ktlint/tree/master/ktlint-ruleset-standard).

I think some pointer on what a `rule-set-id` actually is would help to make it easier to understand the required config keys.
